torch_geometric/nn/conv/message_passing.py

```diff
@@ -612,15 +613,52 @@ def jittable(self, typing: Optional[str] = None):
             prop_types = split_types_repr(match.group(1))
             prop_types = dict([re.split(r'\s*:\s*', t) for t in prop_types])
 
+        # Find and parse `edge_updater` types to format `{arg1: type1, ...}`.
+        # First test whether forward uses `edge_updater`
+        # TODO: Replace with AST search to see if the function is used
+        edge_updater_used = re.search(r'.*edge_updater.*', source)
```

Comment:
I don't think this is very fullproof.. perhaps using AST (as in TODO) would be better.. what do you think @rusty1s ?

Reply:
Possible approach proposed here https://github.com/pyg-team/pytorch_geometric/pull/3776

Reply:
So the problem is that we are unsure whether `edge_updater` is actually used? This shouldn't be a problem when we instantiate `edge_update` with empty arguments by default, right? As an alternative, we could check if `edge_update` is actually overridden.

Reply:
Good point. Again, try to resolve in your below comment.